test/controllers/kafka_controller.py

```diff
@@ -77,11 +80,18 @@ def _get_kafka_env(self):
             "KAFKA_DELETE_TOPIC_ENABLE": "true"
         }
 
+    def delete_topics(self, *topics: str):
+        self._admin.delete_topics(topics)
+
+    def delete_all_topics(self):
+        topics = self._admin.list_topics()
+        topics = [t for t in topics if not t.startswith("_")]
```

Comment:
what's the special case about a topic starting with a `_`?

Reply:
Kafka uses internal topics to manage its partition pointers I believe. In any case, when I naively pulled all the topics and tried to delete them Kafka threw an error

Reply:
Will it delete topics not originated from CDM? We will have our own Kafka instance?

Reply:
This only deletes topics for the docker container the class started